### PR TITLE
Fixed typos and added reference to docker-compose installation.

### PR DESCRIPTION
**Pull request by nisevi**
Fixed typos and added a reference to docker-compose installation (mostly for Linux users).

### CODE REVIEW

File: 3-docker-compose/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 While docker allows you to easily build and link different apps with different environments and runtimes, doing so with more than just two or three containers quickly becomes tedious. `docker-compose` allows you to work with complex container systems very easily.
 
-For working with `docker-compose`, you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
+For working with `docker-compose` you will need to [install it](https://docs.docker.com/compose/install/) depending on your case and you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
```
Comment (gvilarino) [inline 3-docker-compose/README.md]: I'd do something like:

First check if you have `docker-compose` installed (chances are that you have) by doing:

```
docker-compose --version
```

If that fails, [get `docker-compose` for you specific setup](https://docs.docker.com/compose/install/)

**Review by gvilarino — CHANGES_REQUESTED** [review]

**Comment by gvilarino** [comment]
Thanks! I left some comments, please address those and I'll merge.

File: 3-docker-compose/README.md [hunk 2]
```diff
@@ -2,9 +2,17 @@
...
+
+```
+docker-compose --version
+```
+
+If that fails, [get `docker-compose` for you specific setup](https://docs.docker.com/compose/install/)
+
 For working with `docker-compose`, you will need a `docker-compose.yml` file. Check out the one under the `gvilarino/docker-testing` repo.
 
-First make sure you aren't running any containers. The from the project root, do:
```
Comment (nisevi) [inline 3-docker-compose/README.md:7]: @gvilarino I removed the word 'First' here since we are using it on line 5 now.

**Review by gvilarino — APPROVED** [review]
Awsome

**Pull request merged**